### DISCUSSION TITLE
Collecting the BigHash vs BlockCache statistics in Navy

### DISCUSSION THREAD

**Discussion by arungeorge83** [facebook/CacheLib#225] [Q&A]
Hi,
Is there any way to enable BigHash vs BlockCache statistics in Navy on Cachebench runs?
The stat can be like 'the number of items handled by each', 'the flash ops by each' etc.
I see the current output of Cachebench does not log them.

If you can point to a place in code where that can be enabled or added, that also would suffice.
Thanks in advance.

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
